src/connection/index.ts

```diff
@@ -77,6 +81,33 @@ export const walletConnectConnection: Connection = {
   type: ConnectionType.WALLET_CONNECT,
 }
 
+const WALLETCONNECT_PROJECT_ID = process.env.REACT_APP_WALLETCONNECT_PROJECT_ID
+if (typeof WALLETCONNECT_PROJECT_ID === 'undefined') {
+  throw new Error(`REACT_APP_WALLETCONNECT_PROJECT_ID must be a defined environment variable`)
+}
+
+const [web3WalletConnectV2, web3WalletConnectV2Hooks] = initializeConnector<WalletConnectV2>((actions) => {
+  return new WalletConnectV2({
+    actions,
+    options: {
+      chains: NETWORK_SELECTOR_CHAINS,
+      rpcMap: RPC_URLS_WITHOUT_FALLBACKS,
+      projectId: WALLETCONNECT_PROJECT_ID,
+      qrModalOptions: {
+        themeVariables: {
+          '--w3m-z-index': Z_INDEX.modalOverTooltip.toString(),
```

Comment:
Is this necessary if its also in the css file?

Reply:
This is the default way to do it, the CSS file will be gone once this is fixed on Monday and actually works!